components/ReadMore/ReadMore.js

```diff
@@ -0,0 +1,33 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+
+export default function ReadMoreReadLess({ description }) {
+  const [isReadMore, setIsReadMore] = useState(true);
+  const toggleReadMore = () => {
+    setIsReadMore(!isReadMore);
+  };
+  return (
+    <StyledParagraph>
+      {isReadMore ? description.slice(0, 350) : description}
+      <StyledSpan onClick={toggleReadMore}>
+        {isReadMore ? "...read more" : " show less"}
+      </StyledSpan>
```

Comment:
It's good practice to make interactive elements accessible. Instead of using a span for the "read more"/"show less" toggle, you could use a button element to improve accessibility and semantics. Buttons are inherently focusable and support keyboard interactions, which is especially important for screen reader users and those using a keyboard.

Reply:
Nice Features😍 This Comment has more to do with QA, but strange things are happening, when maxing out the character amount c.f.: 
![image](https://github.com/Aytac-Karkin/capstone-project-book-app/assets/133811635/3347fb04-c401-45ff-9900-790fdd17fe20)
i guess this has something to do with your <CommentsList /> container.

Reply:
We´ll look into that!